packages/synthetix-main/contracts/storage/Pool.sol

```diff
@@ -219,10 +219,10 @@ library Pool {
             thing = int(DecimalMath.UNIT); // If minLiquidityRatio is zero, then TODO
```

Comment:
thing... really? LoL

Reply:
Yes, this has to do with the "passes" strategy. Seems like during the last pass I identified that to be a thing, but not what it is. We can probably rename it properly on the next pass.